internal/benchmark/metrics/execution/peers.go

```diff
@@ -20,10 +20,13 @@ const (
 	PeerCountMeasurement = "Count"
 )
 
+var measuringErr = errors.New("UNABLE_TO_MEASURE")
```

Comment:
golang best practice for predefined errors is to store them in separate files like `error.go`

Reply:
I see your point. I would say it's pretty common to declare errors in the file with some other objects. It depends on the size of the file or package, etc. If there are more package level errors, then we can move it to a separate file. Good suggestion